packages/delegate/src/Transformer.ts

```diff
@@ -9,11 +9,11 @@ interface Transformation {
   context: Record<string, any>;
 }
 
-export class Transformer {
+export class Transformer<TContext = Record<string, any>> {
```

Comment:
Is there any benefit in passing the TContext along with the Transforms or does it make more sense to just use any for all the type signatures that complain that TContext might be wrong (e.g. in the constructor)?

Reply:
I guess yes as `DelegationContext``holds information about the Context?

Reply:
We could release an alpha and ask for feedback? But, this would slow down the process of actually releasing this.

Reply:
But since it is just a default value and everyone could overwrite it if desired I guess it is fine to leave it like that.